docs/cpp/enable-logging-cpp.md

```diff
@@ -8,14 +8,13 @@ DateApproved: 07/25/2019
 MetaDescription: How to enable logging in the Visual Studio Code C/C++ extension
 ---
 # C/C++ extension logging
-
-If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
+Logging information is available for the language server and debugger. If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
 
 Logging information is delivered directly to the Output window in Visual Studio Code.
```

Comment:
Need to change this to point to the Output window for the language service and debugConsole for the debugger.